database/src/main/scala/cromwell/database/SqlDatabase.scala

```diff
@@ -10,184 +9,177 @@ import scalaz.NonEmptyList
 
 
 trait SqlDatabase extends AutoCloseable {
-  protected def isTransient(throwable: Throwable): Boolean = {
-    throwable match {
-      case e: SQLTransientException => true
-      case _ => false
-    }
-  }
-
-  protected def createWorkflow(workflowExecution: WorkflowExecution,
+  def createWorkflow(workflowExecution: WorkflowExecution,
```

Comment:
Yes a lot of them are useless now. Although I fee like deleting those implies deleting their Slick implementation which implies deleting the unused tables etc...
I don't think it's in the scope of this ticket. I'll make one.